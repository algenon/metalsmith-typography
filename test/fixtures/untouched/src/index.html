<h1>Beautiful &amp; Awesome Web Typography with “Richtypo”</h1>
<p>Welcome to&nbsp;the&nbsp;world of&nbsp;beautiful web typography&nbsp;&mdash; only with Richtypo. OMG, it's truly awesome, I take it 3-5 times a day.</p>