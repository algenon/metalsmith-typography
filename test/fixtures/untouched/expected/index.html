<h1>Beautiful &amp; Awesome Web Typography with “Richtypo”</h1>
<p>Welcome to the world of beautiful web typography &mdash; only with Richtypo. <abbr>OMG</abbr>, it's truly awesome, I take it <nobr>3-5</nobr> times a day.</p>